docs/topics/testing-rules.adoc

```diff
@@ -61,7 +61,7 @@ The following is an example of a test rule that fails if only a specific message
           id="proprietary-servlet-test" xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
           xsi:schemaLocation="http://windup.jboss.org/schema/jboss-ruleset http://windup.jboss.org/schema/jboss-ruleset/windup-jboss-ruleset.xsd">
   <testDataPath>data/</testDataPath>
-  <rulePath>../proprietary-servlet.mta.xml</rulePath>
+  <rulePath>../proprietary-servlet.windup.xml</rulePath>
   <ruleset>
     <rules>
       <rule id="proprietary-servlet-01000-test">
```

Comment:
line 112 displays  servlet.mta.xml

Reply:
@stillalearner Please check if the changes are implemented.